scripts/tool-install.sh

```diff
@@ -12,5 +12,19 @@ sudo apt-get install pfring nprobe ntopng ntopng-data n2disk cento nbox
 sudo apt-get install wireshark tshark
 sudo apt-get update
 
+#install dependencies for bro
+sudo apt-get install cmake make gcc g++ flex bison libpcap-dev libssl-dev python-dev swig zlib1g-dev
+
+#install C++ Actors Framework, Ignore the KEYEXPIRED 1503492954 or NO_PUBKEY error
+sudo sh -c "echo 'deb http://download.opensuse.org/repositories/devel:/libraries:/caf/xUbuntu_14.04/ /' > /etc/apt/sources.list.d/caf.list"
+sudo apt-get update
+sudo apt-get install -y caf
+
 #install bro
-sudo apt-get install bro
+sudo wget https://www.bro.org/downloads/bro-2.5.1.tar.gz
```

Comment:
The installation commands given below do not install bro tool. Please see the [installation steps](https://www.digitalocean.com/community/tutorials/how-to-install-bro-on-ubuntu-16-04) and add the commands, url reference at the end of the script as comment.